hc5-async-json/src/main/java/com/ok2c/hc5/json/http/JsonResponseConsumers.java

```diff
@@ -15,19 +15,18 @@
  */
 package com.ok2c.hc5.json.http;
 
-import com.fasterxml.jackson.core.type.TypeReference;
```

Comment:
@scr-oath Looks good to me. Is there any chance you could avoid reformatting of the imports?

Reply:
What import layout do you prefer? I generally let the IDE optimize/remove after iterating so that I don't leave stragglers and it rearranges according to sorted order of non-java packages followed by javax then java and finally static imports.

<img width="256" alt="image" src="https://user-images.githubusercontent.com/41922797/167886899-0929d625-cb02-47dc-9a2d-f0fd717204b6.png">

Reply:
Ok - I put them in the same order.